tests/ulyanov_et_al_2016/test_modules.py

```diff
@@ -0,0 +1,136 @@
+import itertools
+
+import pytest
+
+import pytorch_testing_utils as ptu
+import torch
+from torch import nn
+
+from pystiche.image.utils import extract_num_channels
+from pystiche_papers.ulyanov_et_al_2016 import modules
+
+
+def test_SequentialWithOutChannels(subtests):
+    sequential_modules = [nn.Conv2d(3, 3, 1), nn.Conv2d(3, 3, 1)]
+    for out_channel_name in (
+        None,
+        1,
+        "1",
+    ):  # TODO: Rename out_channel_name is not implemented
+        module = modules.SequentialWithOutChannels(
+            *sequential_modules, out_channel_name=out_channel_name
+        )
+
+        with subtests.test("out_channel_name"):
+            if out_channel_name is None:
+                desired_out_channel_name = "1"
+            elif isinstance(out_channel_name, int):
+                desired_out_channel_name = str(out_channel_name)
+            else:
+                desired_out_channel_name = out_channel_name
+
+            assert tuple(module._modules.keys())[-1] == desired_out_channel_name
+
+        with subtests.test("out_channels"):
+            assert module.out_channels == sequential_modules[-1].out_channels
+
+
+def test_join_channelwise(subtests, input_image, style_image):
```

Comment:
I would prefer you use `image_small_0` and `image_small_1` since this has nothing to do with the `style_image`.